baseapp/abci_test.go

```diff
@@ -113,8 +113,8 @@ func TestGetBlockRentionHeight(t *testing.T) {
 
 		tc.bapp.SetParamStore(&mock.ParamStore{Db: dbm.NewMemDB()})
 		tc.bapp.InitChain(abci.RequestInitChain{
-			ConsensusParams: &abci.ConsensusParams{
-				Evidence: &tmprototypes.EvidenceParams{
+			ConsensusParams: &tmproto.ConsensusParams{
```

Comment:
Whats the reason behind all these changes from using abci -> tmproto? I'm assuming `tmproto` dependency has the feature for abci++?

Reply:
These are mainly a port from the main SDK version of the PR. But in any case, this is due to Tendermint v0.37 changes.